addons/account/views/account_move_views.xml

```diff
@@ -530,7 +530,7 @@
                                 string="Send &amp; Print"
                                 attrs="{'invisible':['|', '|', ('state', '!=', 'posted'), ('is_move_sent', '=', True), ('move_type', 'not in', ('out_invoice', 'out_refund'))]}"
                                 class="oe_highlight"
-                                groups="account.group_account_invoice"/>
```

Comment:
There are a few things that are wrong here:
* `account` doesn't depend on `sales_team`. So the group `sales_team.group_sale_salesman` won't exist on databases where only `account` is installed.
* I didn't triple check this, but `sales_team.group_sale_salesman` doesn't seem to imply `account.group_account_invoice`, so the group made for this won't work anymore.
* You can't remove the groups altogether otherwise people that have only read access on the invoices will be able to see the button to send a mail to the client.
* There are two buttons, the other one is right under it, to highlight the button when it is not sent yet. Both buttons should have the same behaviour regarding the access rights.

I see a few possible solutions, none are perfect: 
1. Change the `groups` attribute **in the appropriate module** by adding the appropriate group, but keeping the existing one. This  can be problematic if multiple modules try to change it (because only the last change will be taken into account)
1. Since changing the `groups` attribute can be problematic, one solution could be to add a button with only the appropriate group **but in the appropriate module**. This means some users will have both buttons
1. Remove the `groups` attribute, and check the access rights in the method `action_invoice_sent`, and inherit the method **in the appropriate module**. This means some users might see a button but they will get an access error when clicking on it.

The access rights should not only be checked in the view, but in the code execution too; this doesn't seem to be done yet, so a malicious user (with readonly access on `account.move` could send a xml_rpc request to launch the action, and send the mail)

With all this, I think the last proposed solution is the least bad. I would not be surprised if someone re-adds it in a few months not knowing this discussion, so maybe add a comment in the code. (if I understood correctly, we have guidelines to tests the UI more thoroughly in the future regarding the access rights)